src/Components/Onboarding/Steps/Artists/ArtistSearchResults.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import { artsyNetworkLayer } from "../../../../Relay/config"
+import ArtistSearchQueryConfig from "../../../../Relay/Queries/ArtistSearchResults"
+
+import SelectableItemContainer from "./SelectableItemContainer"
+
+export interface RelayProps {
+  searchResults: any[]
+}
+
+export default class ArtistSearchResults extends React.Component<any, any> {
+  render() {
+    return <ArtistSearchResultsContentList />
+  }
+}
+
+function ArtistSearchResultsContentList() {
+  Relay.injectNetworkLayer(artsyNetworkLayer())
```

Comment:
No need to inject a new network layer, this is done _once_ when the page/app is initialized.